app/src/main/java/org/jellyfin/androidtv/model/itemtypes/Episode.kt

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.androidtv.model.itemtypes
+
+import android.graphics.Bitmap
+
+data class Episode (
+	val id: String,
+	val canDelete: Boolean,
+	val canDownload: Boolean,
+	//TODO: Chapters: ArrayList<ChapterInfoDto>
+	val communityRating: Float,
+	val name: String,
+	val overview: String,
```

Comment:
I think we should use our own naming and I prefer "description" instead of "overview"